### DISCUSSION TITLE
[arch linux] Set sddm theme from terminal (scripting)

### DISCUSSION THREAD

**Discussion by Davetheraveuk** [lxqt/lxqt#2160] [Q&A] [answered]
Hi, I would like to be able to set a sddm theme from the arch linux terminal for scripting purposes.
How do I do this? Thanks, David.

**Comment by stefonarch** [comment, marked as answer]
Echoing some theme name to ` /etc/sddm.conf` or replace the line using `sed` ecc
...
[Theme]
Current=theme_name
...
